.travis.yml

```diff
@@ -1,9 +1,13 @@
+sudo: required
+
 language: java
 
 branches:
   only:
     - master
 
+jdk:
+  - openjdk8
```

Comment:
could as well start building this app from java 11 (or whichever is available in azure). for separate pr, of course